src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/settings/SettingsPluginComponent.kt

```diff
@@ -48,6 +54,10 @@ class SettingsPluginComponent {
             .addComponent(showCoverageCheckBox, 10)
             .addComponent(JXTitledSeparator("Environment settings"), 15)
             .addLabeledComponent(JBLabel("Java 11 path:"), javaPathTextField, 10, false)
+            // Add accessibility options
+            .addComponent(accessibilitySeparator)
+            .addComponent(JBLabel("Choose color for visualisation highlight"))
```

Comment:
You should add an inset to those two components in order to keep them consistent with the rest of the UI - 15 for the separator and 10 for the label.

Reply:
did that